app/assets/stylesheets/config/_colors.scss

```diff
@@ -17,8 +17,8 @@
   --base-90: #202428;
   --base-80: #363d44;
   --base-70: #4d5760;
-  --base-60: #64707d;
-  --base-50: #7d8a97;
+  --base-60: #0a0a0a;
+  --base-50: #5d6770;
```

Comment:
How did you come up with these two colors?

All `--base-*` variables are calculated steps/shades of grey. Higher the value, darker the shade. With this change `--base-60` becomes almost the darkest shade of all base colors and whole hierarchy becomes useless.

Instead of changing values of `--base` variables, I'd rather update places where this value is used. 

Maybe you could bring that report in, so I can track these places and fix it.

Reply:
@ludwiczakpawel I tried with the online Ligthouse but it doesn't address Accessibility, or at least I wasn't able to get it to work.

I could reproduce it by opening Chrome on our homepage and then in the developers tools clicking on Audits:

![Screenshot 2020-03-31 at 2 37 44 PM](https://user-images.githubusercontent.com/146201/78027269-644ad500-735d-11ea-8365-5dce188302bb.png)

and then you get this result:

![Screenshot 2020-03-31 at 2 38 35 PM](https://user-images.githubusercontent.com/146201/78027289-6b71e300-735d-11ea-9d50-a05206da8fda.png)

Reply:
Thanks, I'll play with it. But let's please don't merge these changes like that :P